src/Microsoft.ML.TensorFlow/TensorFlow/Tensor.cs

```diff
@@ -446,6 +446,104 @@ internal static unsafe TFTensor CreateString(byte[] buffer)
             return new TFTensor(handle);
         }
 
+        internal static unsafe byte[] DecodeString(TFTensor tensor)
```

Comment:
I don't see calls for this class.

Reply:
It will be called when we need to support scalars. I would like to keep it here if you don't mind...:)

---
In reply to: [256945747](https://github.com/dotnet/machinelearning/pull/2546#discussion_r256945747) [](ancestors = 256945747)